testsuite/tests/performance/README.md

```diff
@@ -71,16 +78,9 @@ When you configured hyperfoil, you need to configure the testsuite. Steps can be
 **WARNING**
 
 You are running the performance tests, which can generate multiple requests per second.
-Keep in mind that you need to choose your own backend. Otherwise, your actions can be considered as a DDoS attack.
-
-Performance tests are only using the primary backend.
-    
-```yaml
-threescale:
-  service:
-    backends:
-      primary: http://httpbin.httpbin.svc:8080
-``` 
+Keep in mind that you need to choose proper backend. Otherwise, your actions can be considered as a DDoS attack.
```

Comment:
I would change the wording to indicate better you should use your backend deployment.

`Keep in mind that you need to choose your own deployments of backends.`